### DISCUSSION TITLE
Chromium Notifications Show raw HTML

### DISCUSSION THREAD

**Discussion by lifeashansen** [Aylur/ags#479] [Q&A]
![image](https://github.com/Aylur/ags/assets/75609068/0a6efb0f-5941-4b01-a152-fdda79af41d7)

Notifications from any chromium based browser from chromium,google chrome , vivaldi and even brave show notifications with the browser icon and some raw html script holding the link instead of a formatted preview text and an icon for the corresponding web app .Firefox notifications work just fine but I cant use it because it runs in a very skibidi way on my computer.

I have tried installing all these browsers from different aurs  , pacman and even snapd but the issue is still persistent so it has to be sth to do with how the notification daemon is rendering notifications from any chromium browser

Does anyone know how to fix this

**Comment by OBLIXi-ON** [comment]
You may need to filter it. Afaik notification `body` by default doesn't use markup. I had the same issue and tried to resolve it with pango but pango doesn't support link tag.

**Reply by lifeashansen** [reply]
> You may need to filter it. Afaik notification `body` by default doesn't use markup. I had the same issue and tried to resolve it with pango but pango doesn't support link tag.

I dont understand...whats afaik

**Comment by Jas-SinghFSU** [comment]
Try something like this
```
            Widget.Label({
                use_markup: true,
                label: notif.body,
            }).on('realize', (self) => {
                self.set_markup(notif.body);
            }),
```
Where the Label is the one displaying the HTML.

**Reply by lifeashansen** [reply]
> Try something like this
> 
> ```
>             Widget.Label({
>                 use_markup: true,
>                 label: notif.body,
>             }).on('realize', (self) => {
>                 self.set_markup(notif.body);
>             }),
> ```
> 
> Where the Label is the one displaying the HTML.

Can I share my dots repo link and you can guide me on exactly what I should change

https://github.com/whoslucifer/nix.git
